tests/testthat/test-checkers.R

```diff
@@ -0,0 +1,42 @@
+test_that("is_parameterised works as expected for epiparam", {
+  res <- is_parameterised(epiparam())
+  expect_length(res, 118)
```

Comment:
Could this be not hardcoded? It will fail when more parameters are added to the package library

Reply:
Its not possible to remove all hardcoding as updating the library may break the test. I've reduced the use of magic numbers in 240c42f882ed9063c87b3e2b00f5f20e1cda0159.